lib/cartopy/mpl/geoaxes.py

```diff
@@ -270,6 +284,27 @@ def __init__(self, *args, **kwargs):
         self.img_factories = []
         self._done_img_factory = False
 
+    def _init_axis(self):
+        """
+        Initialize Axis variables
+
+        GeoAxes does not use the Axis for ticking or labelling, though
+        Matplotlib still uses it internally. So we need to keep it around, but
+        we disable the ticks using these small wrapper classes. We also do not
+        bother to register the Axis with any Spines.
+        """
+        self.xaxis = _TicklessXAxis(self)
+        self.yaxis = _TicklessYAxis(self)
+
+    def _gen_axes_spines(self, locations=None, offset=0.0, units='inches'):
```

Comment:
@QuLogic this method appears to be a duplicate of the one at [L1195-L1204](https://github.com/QuLogic/cartopy/blob/8a17a897f2dbb8d071803cb0806d15217cdbdde0/lib/cartopy/mpl/geoaxes.py#L1195-L1204)?